apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerTransaction.cs

```diff
@@ -58,6 +58,13 @@ public sealed class SpannerTransaction : DbTransaction, ISpannerTransaction
 
         private Session Session { get; }
 
+        // Note: We use seconds here to follow the convention set by DbCommand.CommandTimeout.
```

Comment:
Ack. Maybe add the meaning of 0 here.
(I know it's going to be annoying to copy it all over the place, but it'll definitely make users' lives easier :)